ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -36,6 +50,13 @@ FAN_SS_VSC:
   - FAN
   - SS
   - VSC
+ 
+FAN_DFMSC:
```

Comment:
unless there is a specific reason to use discharge fan in the field names, none of the fields should include them. might require you to make a new type called MSC, which you can copy from DFMSC and remove `discharge_fan_` from the fields.

Reply:
removed the canonical....